draft-irtf-cfrg-vdaf.md

```diff
@@ -1247,16 +1247,14 @@ remainder of this document. Note that an exception is raised by each function if
 the operands are not the same length.
 
 ~~~
-# Compute the inner product of the operands.
-def inner_product(left: Vec[Field], right: Vec[Field]) -> Field:
-    return sum(map(lambda x: x[0] * x[1], zip(left, right)))
```

Comment:
This function is not used anywhere, and no longer appears in the proof of concept implementation, so I deleted it rather than update it.